src/raylib.h

```diff
@@ -1334,6 +1334,7 @@ RLAPI Image ImageCopy(Image image);
 RLAPI Image ImageFromImage(Image image, Rectangle rec);                                                  // Create an image from another image piece
 RLAPI Image ImageText(const char *text, int fontSize, Color color);                                      // Create an image from text (default font)
 RLAPI Image ImageTextEx(Font font, const char *text, float fontSize, float spacing, Color tint);         // Create an image from text (custom sprite font)
+RLAPI Image ImageMaskFromImage(Image image, Color color, float threshold);                               // Create an image mask from an image (alpha channel)
```

Comment:
I think this function could be just `ImageFromChannel()` and return selected channel as a GRAYSCALE/R32 image.

Reply:
Can't find ImageFromChannel in the master branch...

Reply:
@brccabral It does not exist, it's a proposed more general alternative to adding `ImageMaskFromImage()`

Reply:
Got it. Close this PR whenever you want to. Looking forward to ImageFromChannel. Saludos